test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -26,6 +26,7 @@
 using Newtonsoft.Json;
 using Xunit;
 using Xunit.Abstractions;
+using System.Diagnostics;
```

Comment:
>Diagnostics [](start = 13, length = 11)

Is this used? If it is, please move it up where the rest of the System.* usings are.